components/StyledComponents/LinkWithoutDecoration.js

```diff
@@ -0,0 +1,6 @@
+const { default: Link } = require("next/link");
```

Comment:
That's a cumbersome way of importing the `Link` component 😆  Why don't you take this?
```suggestion
import Link from "next/link";
```